src/snowflake/snowpark/modin/pandas/__init__.py

```diff
@@ -171,6 +178,7 @@
 import snowflake.snowpark.modin.plugin.extensions.base_overrides  # isort: skip  # noqa: E402,F401
 import snowflake.snowpark.modin.plugin.extensions.dataframe_extensions  # isort: skip  # noqa: E402,F401
 import snowflake.snowpark.modin.plugin.extensions.dataframe_overrides  # isort: skip  # noqa: E402,F401
+import snowflake.snowpark.modin.plugin.extensions.general_overrides  # isort: skip  # noqa: E402,F401
```

Comment:
you can delete this import here for now since you're explicitly importing all of the methods with `from snowflake.snowpark.modin.plugin.extension.general_overrides import (...)`

the other modules need to be imported because their members are registered with the `register_pd_accessor` decorator